packages/gatsby/src/internal-plugins/dev-404-page/raw_dev-404-page.js

```diff
@@ -78,10 +78,16 @@ class Dev404Page extends React.Component {
   }
 
   render() {
+    const functionsEnabled =
+      this.props.data.allSiteFunction.nodes[0].url !== `FAKE`
```

Comment:
What's up with this `FAKE`? Add a comment to explain why it's there

Reply:
added comment